lm_eval/models/openai_completions.py

```diff
@@ -226,6 +227,11 @@ def api_key(self):
         key = os.environ.get("OPENAI_API_KEY", None)
         if key is None:
             raise ValueError(
-                "API key not found. Please set the OPENAI_API_KEY environment variable."
+                "API key not found. Please set the `OPENAI_API_KEY` environment variable."
             )
         return key
+
+    def loglikelihood(self, requests, **kwargs):
+        raise NotImplementedError(
+            "Loglikelihood (and multiple-choice) is not supported for chat completions as OpenAI does not provide logprobs."
```

Comment:
```suggestion
            "Loglikelihood (and therefore `multiple_choice`-type tasks) is not supported for chat completions as OpenAI does not provide prompt logprobs. See https://github.com/EleutherAI/lm-evaluation-harness/issues/942#issuecomment-1777836312 or https://github.com/EleutherAI/lm-evaluation-harness/issues/1196 for more background on this limitation."
```

Reply:
Perhaps we can make this error message verbose enough to minimize how many new people need to ask clarification on it?